.circleci/config.yml

```diff
@@ -1987,8 +1998,10 @@ workflows:
       - t_ubu_cli: *requires_b_ubu
       - t_ubu_locale: *requires_b_ubu
       - t_ubu_soltest_all: *requires_b_ubu
+      - t_ubu_arm_soltest_all: *requires_b_ubu_static_arm
       - b_ubu_clang: *requires_nothing
       - t_ubu_clang_soltest: *requires_b_ubu_clang
+      - t_ubu_arm_soltest_all: *requires_b_ubu_static_arm
```

Comment:
You have soltest twice and are missing CLI tests.

Reply:
Fixed.